### PR TITLE
Merged Cross-browser-compatible correspondence function changes to root by comparing with C# version.

### PR DESCRIPTION
**Pull request by supragyan**
Merged below files to root for VB version by comparing with C# version.
- master pages
- js pages

### CODE REVIEW

**Comment by daisukenishino** [comment]
@supragyan 

There is a problem with Global.asax.
This does not separate as the code-behind.

https://github.com/OpenTouryoProject/OpenTouryoTemplates/blob/master/root_VS2010/programs/C%23/Samples/WebApp_sample/ProjectX_sample/Global.asax

File: root/programs/VB/Samples/WebApp_sample/ProjectX_sample/Global.asax [hunk 1]
```diff
@@ -1 +1,282 @@
-﻿<%@ Application Codebehind="Global.asax.vb" Inherits="ProjectX_sample.Global_asax" Language="vb" %>
```
Comment (daisukenishino) [inline root/programs/VB/Samples/WebApp_sample/ProjectX_sample/Global.asax]: Don't delete this line.

**Comment by supragyan** [comment]
@daisukenishino 

We have modified as above for root in this pull request.

Please confirm.

**Comment by daisukenishino** [comment]
@supragyan 

I wanted a separation of code from the previous.
Thank you proposed this, Please do so.

**Pull request merged**